frontend/src/metabase/nav/containers/MainNavbar/SidebarItems/SidebarItems.styled.tsx

```diff
@@ -106,10 +111,20 @@ export const FullWidthButton = styled.button<{ isSelected: boolean }>`
       color: var(--mb-color-brand);
     }
   }
+
+  &:focus,
+  &:focus-visible {
+    outline: none;
+  }
 `;
 
 export const FullWidthLink = styled(Link)`
   ${itemContentStyle}
+
+  &:focus,
+  &:focus-visible {
+    outline: none !important;
```

Comment:
I'm the first person to dislike `!important` but it simply couldn't work without it. Styled `Link` was overriding the styles we apply here.

In general, this whole charade of styled components used in the main sidebar should be burned to the ground, and then we start fresh. But I'm afraid that's outside the scope of this PR.